language-support/ts/daml-ledger/README.md

```diff
@@ -33,40 +33,38 @@ Please consult the
 [documentation](https://docs.daml.com/app-dev/bindings-ts/daml-ledger/classes/_index_.ledger.html)
 for their exact signatures.
 
-`create`
---------
+## `create`
```

Comment:
It looks like this was previously formatted as an RST document. This PR is already huge and if you have further work that you have done while addressing the formatting it should go in a separate PR to give it the proper attention (in a PR that formats a lot of code, this can easily go unnoticed).

Reply:
I just noticed that this happened to all MarkDown files, so I assume this is performed automatically by `prettier`, so nothing contrary to merging those as part of this PR.